pytrip/concave_tool.py

```diff
@@ -0,0 +1,302 @@
+import math
+
+# This file contains methods that create from a list of intersections (list of points).

```

Comment:
I propose to move this file to `res` module under a name `contour_intersection.py`